docker/Dockerfile.k8s

```diff
@@ -15,6 +15,7 @@ RUN pip --no-cache-dir install --root /tmp/dependencies -r requirements.txt
 FROM python:2.7-slim as scalyr
 MAINTAINER Scalyr Inc <support@scalyr.com>
 
+ENV SCALYR_STDOUT_SEVERITY ERROR
```

Comment:
This could be a little cryptic.  Let's add a comment here saying `Configure agent to only emit log messages with ERROR and above to stdout by default`